htdocs/core/lib/functions.lib.php

```diff
@@ -7047,7 +7043,7 @@ function getCommonSubstitutionArray($outputlangs, $onlykey = 0, $exclude = null,
 			} else {
 				// Set the online payment url link into __ONLINE_PAYMENT_URL__ key
 				require_once DOL_DOCUMENT_ROOT.'/core/lib/payments.lib.php';
-				$outputlangs->loadLangs(array('paypal', 'other'));
+				$langs->loadLangs(array('paypal', 'other'));
```

Comment:
$langs is language of logged user.
The language we want to make translation to is   $outputlangs  received as parameter. We must use this one.